app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, file, path = command.split()
+
+    if not command_name == "mv" or file == path:
+        return
+
+    path_to_file, file_from_path = os.path.split(path)
+    if len(path.split("/")) > 1:
```

Comment:
Make your code cross-platform, this check won't work with `\`

Reply:
Oh, I forgot to update the last changes, I have done that